### PR TITLE
Manage certs through Ansible

### PR DESCRIPTION
**Pull request by KellerFuchs**
Not nearly as good as having per-service, automatically-acquired certs from Let's Encrypt, but nobody found the time to do that so far...

Thanks a bunch to wxs4274 for noticing that `mail.#!` had a very outdated certificate.

### CODE REVIEW

**Comment by KellerFuchs** [comment]
This is very not DRY, so if you have ideas on how to refactor it, please say so  :)

File: certs.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- hosts: irc_servers
+  become: False
+  vars_files:
+    - files/certs/hashbang.sh.yml
+    - files/certs/hashbang.sh.key
+  tasks:
+    - copy:
+        content: "{{ item.value }}"
+        dest: "unrealircd/certs/{{ item.key }}"
+        mode: 0400
```
Comment (RyanSquared) [inline certs.yml:37]: Is there a reason this specific one is `0400`?

Comment (KellerFuchs) [reply certs.yml:37]: I kept the permissions that were already there.

**Comment by KellerFuchs** [comment]
~This will need a fix once #85 lands, since `files/certs/hashbang.sh.key` is encrypted to the current vault key...~

File: certs.yml (hunk 1 above)
Comment (RyanSquared) [reply certs.yml:37]: Alright, guess there's no issue with that.

**Review by DeviaVir — APPROVED** [review]
I think this is as DRY as it gets with ansible, unless you start wrapping ansible ;)

**Pull request merged**